src/step-input.scss

```diff
@@ -0,0 +1,153 @@
+@import "./settings";
+@import "./new-settings";
+@import "./mixins";
+@import "./functions";
+@import "./button";
+
+$block: #{$fd-namespace}-step-input;
+
+.#{$block} {
+  $fd-step-input-border-thin-width: 0.0625rem;
+  $fd-step-input-border-thick-width: 2 * $fd-step-input-border-thin-width;
+  $fd-input-group-button-active-focus-color: var(--sapContent_ContrastFocusColor);
+  $fd-button-icon-font-size: 0.875rem;
+  $fd-button-icon-font-size-cozy: 1rem;
+
+  @mixin iconSize($size) {
+    &::before,
+    &::after {
+      font-size: $size;
+    }
+  }
+
+  width: auto;
+  display: inline-block;
+  line-height: 0;
+
+  @include fd-fiori-focus() {
+    outline-color: $fd-input-group-button-active-focus-color;
+  }
+
+  @include fd-reset();
+  @include fd-rtl();
+
+  &__content-wrapper {
+    width: auto;
+    min-width: 2.75rem;
+    margin-top: 0.25rem;
+    margin-bottom: 0.25rem;
+    padding: 0;
+    box-sizing: border-box;
+    vertical-align: middle;
+    display: inline-block;
+    position: relative;
+
+    &.is-readonly {
+      outline: none;
+    }
+  }
+
+  &__content-wrapper:focus-within {
+    box-shadow: none;
+    outline-offset: -0.1875rem;
+    outline-width: var(--sapContent_FocusWidth);
+    outline-color: var(--sapContent_FocusColor);
+    outline-style: var(--sapContent_FocusStyle);
+
+    &.is-readonly {
+      outline: none;
+    }
```

Comment:
Same for this section? Is this specific to step input?

Reply:
Now that I play with it more, a bunch of the focus stuff needs to go in general. Try using this component with your keyboard - it's not possible to tab to the buttons as currently written.